From 552e6b6c1065278299c00530fe2af9e280ea9ebc Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Thu, 16 Mar 2023 19:33:03 +0530
Subject: [PATCH 119/500] OpenWrt: 261-lib-arc4-unhide.patch

lib/crypto: add tristate string for ARC4

This makes it possible to select CONFIG_CRYPTO_LIB_ARC4 directly. We
need this to be able to compile this into the kernel and make use of it
from backports.

Signed-off-by: Felix Fietkau <nbd@nbd.name>

Change-Id: Ibac6150238888a48a2b6d793b704b6bb9efb50c3
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
(cherry picked from commit 1f24244080848d04aaaf86a900bb56c538b98cfb)
---
 lib/crypto/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/crypto/Kconfig b/lib/crypto/Kconfig
index 45436bfc6dff..4f7147925cf3 100644
--- a/lib/crypto/Kconfig
+++ b/lib/crypto/Kconfig
@@ -15,7 +15,7 @@ config CRYPTO_LIB_AESGCM
 	select CRYPTO_LIB_UTILS
 
 config CRYPTO_LIB_ARC4
-	tristate
+	tristate "ARC4 cipher library"
 
 config CRYPTO_LIB_GF128MUL
 	tristate
-- 
2.34.1

